api/deployments/models/component_deployment.go

```diff
@@ -450,7 +474,10 @@ func GetReplicaSummary(pod corev1.Pod) ReplicaSummary {
 	replicaSummary.Image = containerStatus.Image
 	replicaSummary.ImageId = containerStatus.ImageID
 	if len(pod.Spec.Containers) > 0 {
-		replicaSummary.Resources = ConvertResourceRequirements(pod.Spec.Containers[0].Resources)
+		replicaSummary.Resources = pointers.Ptr(ConvertResourceRequirements(pod.Spec.Containers[0].Resources))
+	}
+	if len(replicaSummary.StatusMessage) == 0 && (replicaSummary.Status.Status == Failing.String() || replicaSummary.Status.Status == Pending.String()) {
+		replicaSummary.StatusMessage = lastEventWarning
```

Comment:
Not sure its a good idea to set last warning event in StatusMessage. We should instead discuss if we shall show events per component and/or replica, as described in my other comment

Reply:
Same reason as [above](https://github.com/equinor/radix-api/pull/603#discussion_r1540966857) - to keep the failure reason in the radix-batch status, when replicas and events do not longer exist